packages/mui-envinfo/README.md

```diff
@@ -39,3 +39,9 @@ $ npx @mui/envinfo
     styled-components:  5.2.1
     typescript: ^4.0.2 => 4.0.5
 ```
+
+## Options
+
+`--json` - writes output as JSON
+
+`--skipClipboard` - doesn't copy information to clipboard
```

Comment:
Personally, I would expect the opposite (opt-in, instead of opt-out) or to only do it if my clipboard is empty. I would hate it if it was replacing what I saved in it.

Reply:
yes an opt-in is also possible. I'll update the PR to make it opt-in. As long as it makes copying the info easier

> To make sure it doesn't slowdown the execution of the command too much.

locally on my machine ( OSX ) it runs pretty fast

Reply:
Another idea: maybe we could have a prompt to ask developers to accept or reject the copy to clipboard action. This would have the advantage of solving the affordability issue. Otherwise, how would a developer know that it's in his clipboard?

Reply:
> Otherwise, how would a developer know that it's in his clipboard?

Maybe you missed "Output copied to the clipboard" in the screen above 😉 - there is also a message for a failure